python/lsst/daf/butler/cli/cmd/commands.py

```diff
@@ -599,3 +599,31 @@ def ingest_files(**kwargs):
     raw camera exposures.
     """
     script.ingest_files(**kwargs)
+
+
+@click.command(cls=ButlerCommand)
+@repo_argument(required=True)
+@click.argument("dataset_type", required=True)
+@click.argument("storage_class", required=True, nargs=1)
```

Comment:
Why `nargs=1` here, but not above? Maybe I'm misinterpreting the `click.argument` docs, but they suggest that passing `nargs` makes the result a tuple?